Formula/scala@2.10.rb

```diff
@@ -37,7 +37,10 @@ def install
     doc.install Dir["doc/*"]
     share.install "man"
     libexec.install "bin", "lib"
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    Dir["#{libexec}/bin/*"].each do |file|
```

Comment:
why aren't we using env_script_all_files?

Reply:
The tests fail when I try to do it that way. I have no idea why.

Reply:
Okay, I think I've got it figured out. Amended.

Reply:
God help me, I've been writing code for near 20 years now, and the Homebrew codebase is one of the most difficult ones I've worked with.